test/Driver/toolchain_darwin.swift

```diff
@@ -0,0 +1,9 @@
+// RUN: %swiftc_driver -v -tools-directory %S/Inputs/fake-toolchain %s 2>&1 | %FileCheck -check-prefix LDSUB %s
+
+// REQUIRES: objc_interop
```

Comment:
Please don't use this for tests that aren't actually about ObjC. `REQUIRES: OS=macosx` is close enough.